spartan/aztec-network/values/release.yaml

```diff
@@ -0,0 +1,68 @@
+network:
+  public: true
+
+images:
+  aztec:
+    pullPolicy: Always
+
+telemetry:
+  enabled: true
+  otelCollectorEndpoint: http://34.150.160.154:4318
```

Comment:
What deploy flow does this use?

Reply:
The metrics stack? I ran it one-off from `spartan/metrics/deploy-prod.sh` 😬